app/controllers/facility_reservations_controller.rb

```diff
@@ -149,8 +149,8 @@ def update_admin
     raise ActiveRecord::RecordNotFound unless @reservation.order_detail_id.nil?
     set_windows
 
-    @reservation.assign_times_from_params(params[:reservation])
-    @reservation.admin_note = params[:reservation][:admin_note]
+    @reservation.assign_times_from_params(params[:admin_reservation])
```

Comment:
Sorry, looking at the wrong action. I am unable to create an admin reservation. Looks like `create` isn't using `assign_times_from_params`.